idseq_dag/steps/run_srst2.py

```diff
@@ -32,31 +33,33 @@ def run(self):
                 PipelineStepRunSRST2.fill_file_path(f)
         else:
             # Post processing of amr data
+            self.generate_mapped_reads_table()
```

Comment:
At line 32, should that be changed to `[2:6]`? Since you now have an additional output file.

Reply:
Yes, good catch!